woocommerce/woocommerce
# fix: ValidatedTextInput  attribute

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes https://github.com/woocommerce/woocommerce/issues/51792



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

- Build the project (`pnpm run build`)
- [Ensure your checkout page uses the checkout block
](https://woocommerce.com/document/woocommerce-store-editing/customizing-cart-and-checkout/#using-the-cart-and-checkout-blocks)
- Add a product to the cart (e.g.: the "Album" product from the product templates
- Go to the checkout page
- Don't fill up all the fields - leave last name empty, for example
- Try to submit the checkout form by clicking "Place order"
- An error should appear below the empty fields
- The error message should have an `id` attribute, whose value is referenced on the respective input, which also has the `aria-errormessage` attribute with the same value as the `id` attribute on the error message
<img width="913" alt="Screenshot 2024-10-09 at 10 05 03 AM" src="https://github.com/user-attachments/assets/93313562-b931-428f-a486-4a3d272d6e5a">




### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message fix: ValidatedTextInput aria-errormessage matching

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
